docs/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -137,7 +172,28 @@ spec:
 ```
 
 ### Creating Service by specifying a floating IP
-TBD
+If you already have a floating network and want to specify a floating ip for loadbalance service.
+You need to set `loadBalancerIP` in service spec. Here is an example below.
+```shell
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.kubernetes.io/openstack-internal-load-balancer: "false"
+    loadbalancer.openstack.org/floating-network-id: "9be23551-38e2-4d27-b5ea-ea2ea1321bd6"
+  name: nginx-internet
+spec:
+  ports:
+  - port: 80
+    targetPort: 80
+  loadBalancerIP: 122.112.219.229
+  selector:
+    app: nginx
+  type: LoadBalancer
+```
+It will find a floating ip with `122.112.219.229` and check if it is used. If the floating ip is existed and not bind any ports. Your loadbalancer will bind it.
```

Comment:
Suggestion: Please make sure `122.112.219.229` is available or not allocated, otherwise the Service creation will fail.

Reply:
done